gravatar-quickeditor/src/main/java/com/gravatar/quickeditor/ui/extensions/QESnackbar.kt

```diff
@@ -0,0 +1,81 @@
+package com.gravatar.quickeditor.ui.extensions
```

Comment:
Nitpicking here and I don't really mind it, so feel free to ship it like that ;)

I think we haven't prefixed component names in other places and IMO we shouldn't do it (so I'd drop `QE` here).

Reply:
I started prefixing those that have clashing names. I'm not the biggest fan when you get multiple options with the same name in the autocomplete popup. 

This one is just for the file name so I can drop it - but I did it for some Composables in other PRs.

Reply:
I don't think it's bad to prefix it especially since they're `internal` so it's just for our convenience.